parser/src/main/scala/aqua/parser/expr/PushToStreamExpr.scala

```diff
@@ -0,0 +1,21 @@
+package aqua.parser.expr
+
+import aqua.parser.Expr
+import aqua.parser.lexer.Token._
+import aqua.parser.lexer.{Value, VarLambda}
+import aqua.parser.lift.LiftParser
+import cats.Comonad
+import cats.parse.{Parser => P}
+
+case class PushToStreamExpr[F[_]](
+  stream: VarLambda[F],
```

Comment:
You just need to read `Name[F]`, as you can't write to a stream with lambda. E.g. `stream! <<- "val"` is incorrect.